### PR TITLE
Convert to Swift 2.0 syntax

### PR DESCRIPTION
**Pull request by sdduursma**

### CODE REVIEW

File: OAuthSwift/OAuthSwiftHTTPRequest.swift [hunk 1]
```diff
@@ -70,10 +70,15 @@ public class OAuthSwiftHTTPRequest: NSObject, NSURLConnectionDataDelegate {
...
+            do {
+                self.request = try OAuthSwiftHTTPRequest.makeRequest(self.URL, method: self.HTTPMethod, headers: self.headers, parameters: self.parameters, dataEncoding: self.dataEncoding, encodeParameters: self.encodeParameters, body: self.HTTPBodyMultipart, contentType: self.contentTypeMultipart)
+            } catch let error1 as NSError {
+                error = error1
+                self.request = nil
+            }
                 
             if ((error) != nil) {
-                println(error!.localizedDescription)
+                print(error!.localizedDescription)
```
Comment (rvangraan) [inline OAuthSwift/OAuthSwiftHTTPRequest.swift]: I think you should get rid of the catch, delete the print and add a throws

File: OAuthSwift/OAuthSwiftHTTPRequest.swift [hunk 2]
```diff
@@ -70,10 +70,15 @@ public class OAuthSwiftHTTPRequest: NSObject, NSURLConnectionDataDelegate {
         if (request == nil) {
             var error: NSError?
             
-            self.request = OAuthSwiftHTTPRequest.makeRequest(self.URL, method: self.HTTPMethod, headers: self.headers, parameters: self.parameters, dataEncoding: self.dataEncoding, encodeParameters: self.encodeParameters, error: &error, body: self.HTTPBodyMultipart, contentType: self.contentTypeMultipart)
+            do {
+                self.request = try OAuthSwiftHTTPRequest.makeRequest(self.URL, method: self.HTTPMethod, headers: self.headers, parameters: self.parameters, dataEncoding: self.dataEncoding, encodeParameters: self.encodeParameters, body: self.HTTPBodyMultipart, contentType: self.contentTypeMultipart)
+            } catch let error1 as NSError {
+                error = error1
+                self.request = nil
```
Comment (rvangraan) [inline OAuthSwift/OAuthSwiftHTTPRequest.swift]: delete this. (The whole catch)

File: OAuthSwift/OAuthSwiftHTTPRequest.swift [hunk 3]
```diff
@@ -140,16 +144,17 @@ public class OAuthSwiftHTTPRequest: NSObject, NSURLConnectionDataDelegate {
...
                             var jsonError: NSError?
-                            if let jsonData: NSData = NSJSONSerialization.dataWithJSONObject(nonOAuthParameters, options: nil, error: &jsonError)  {
+                            do {
+                                let jsonData: NSData = try NSJSONSerialization.dataWithJSONObject(nonOAuthParameters, options: [])
                                 URLRequest.setValue("application/json; charset=\(charset)", forHTTPHeaderField: "Content-Type")
                                 URLRequest.HTTPBody = jsonData
-                            }
-                            else {
-                                if (error != nil) {
+                            } catch let error1 as NSError {
```
Comment (rvangraan) [inline OAuthSwift/OAuthSwiftHTTPRequest.swift]: just delete the `do { } catch`, and add `throws`

File: OAuthSwift/OAuthSwiftHTTPRequest.swift (hunk 1 above)
Comment (sdduursma) [inline OAuthSwift/OAuthSwiftHTTPRequest.swift]: Do you mean add `throws` to `start()` ?

**Comment by sdduursma** [comment]
Not ready for merging yet.

**Comment by sdduursma** [comment]
This should be ready now.

**Pull request merged**